src/test/groovy/life/qbic/datamodel/datasets/datastructure/OxfordNanoporeMeasurementSpec.groovy

```diff
@@ -148,4 +148,41 @@ class OxfordNanoporeMeasurementSpec extends Specification {
 
     }
 
+
+    def "If both fast5 pass and fail folder are empty, an IllegalStateException shall be thrown"() {
```

Comment:
I think at least one test case with one folder existing and the other missing should be added